### PR TITLE
fix #68151: Allowed Edit Style window to remember last user resize

### PR DESCRIPTION
**Pull request by WASamK**

### CODE REVIEW

File: mscore/editstyle.cpp [hunk 1]
```diff
@@ -970,5 +972,39 @@ void EditStyle::resetStyleValue(int i)
       setValues();
       }
 
+//---------------------------------------------------------
+//   closeEvent
+//---------------------------------------------------------
+
+void EditStyle::closeEvent( QCloseEvent* )
```
Comment (Jojo-Schmitz) [inline mscore/editstyle.cpp]: too many spaces

File: mscore/editstyle.cpp [hunk 2]
```diff
@@ -970,5 +972,39 @@ void EditStyle::resetStyleValue(int i)
...
+      writeSettings();
+      }
+
+//---------------------------------------------------------
+//   writeSettings
+//---------------------------------------------------------
+
+void EditStyle::writeSettings()
+      {
+      QSettings settings( "museScore", "editStyle" );
```
Comment (Jojo-Schmitz) [inline mscore/editstyle.cpp]: ditto and on most of the following lines too. Drop the space after ( and before )

**Comment by lasconic** [comment]
Hi, I saw you filed and assign yourself several of similar issues. That's great!
It would be good if we could check how it's done for other dialogs in MuseScore. Also check if/when we use http://doc.qt.io/qt-5/qwidget.html#saveGeometry and when/if not, and if so why or why not. The goal would be to have all dialog using the same strategy, maybe even have 2 util functions where we could pass the Widget and a QString  to save/restore the position.

**Comment by WASamK** [comment]
Hi,
The Start Center implements the "Remember last user resize and last user
position" feature. I will implement this feature in a similar manner to the
other dialogs requested as well.

**Comment by Jojo-Schmitz** [comment]
I think you could add them as separate commits to this PR. After you've fixed the formatting issue of the current commit that is ;-)

**Comment by Jojo-Schmitz** [comment]
Rebase is needed

**Comment by IsaacWeiss** [comment]
@WASamK, if you're still around and interested in contributing, just a note that this issue and the others like it have been fixed: https://github.com/musescore/MuseScore/pull/2752

**Pull request closed without merging**